src/styled.js

```diff
@@ -13,19 +15,23 @@ function styled(tag) {
   return function () {
     const _args = arguments;
 
-    return function Styled(props) {
+    function Styled(props, ref) {
```

Comment:
I also thought of adding `displayName` support, i.e. 
```
Styled.displayName = `styled(${getDisplayName(tag)})`;
``` 
but the bundle size would be an issue again.